libs/core/src/lib/calendar/calendar.component.html

```diff
@@ -28,7 +28,7 @@
             [selectedRangeDate]="selectedRangeDate"
             (selectedRangeDateChange)="selectedRangeDateChanged($event)"
             [currentlyDisplayed]="currentlyDisplayed"
-            [startingDayOfWeek]="startingDayOfWeek"
+            [startingDayOfWeek]="getWeekStartDay()"
```

Comment:
> Nice work! Getting a build error:
> 
> ```
> > nx run moment-adapter:build 
> Cannot convert undefined or null to object
> ```
> 
> And a few other concerns. See comments

I have a problem with moment-adapter build
![image](https://user-images.githubusercontent.com/5276786/126729128-f3db4075-752b-456f-9582-763ceace529f.png)
Unfortunately I can't solve it quick :/ something strange there
"get amount of weeks" are not abstract, and DateLocale already at export, at other - fixed, thanks a lot for feedback!

Reply:
> We try to avoid binding to functions in the templates for performance reasons - better to bind to a property if possible, or subscribe to changes on an observable

moved this change request to the separated ticket:
https://github.com/SAP/fundamental-ngx/issues/6116